tests/test_safari_bigint64array_shim.py

```diff
@@ -0,0 +1,154 @@
+import json
```

Comment:
This test doesn't look consistent with our coding standards - it uses 4 spaces, for example.

But, could it not be a test in `test_other.py`? With the large inline code parts separated into separate files perhaps?